src/test/java/plugins/SshSlavesPluginTest.java

```diff
@@ -99,4 +129,20 @@ private void verify() {
         job.save();
         job.startBuild().shouldSucceed();
     }
+    
+    private void verifyLog(String message) {
+        elasticSleep(3000);
```

Comment:
despite its name elasticSleep is not and this will pause all tests for 3 seconds.
Better to use waitFor(... 3) which will wait 3 seconds for the log to contain the matcher that you want, but in the fast case will return as soon as the data is written.